tools/shared/src/main/scala/org/scalajs/core/tools/javascript/Trees.scala

```diff
@@ -61,7 +61,12 @@ object Trees {
     def ref(implicit pos: Position): Tree = VarRef(name)
   }
 
-  case class ParamDef(name: Ident)(implicit val pos: Position) extends Tree {
+  /** ES6 let or const (depending on the mutable flag). */
+  case class Let(name: Ident, mutable: Boolean, rhs: Tree)(implicit val pos: Position) extends Tree {
+    def ref(implicit pos: Position): Tree = VarRef(name)
+  }
```

Comment:
It turns it is used in a couple places. I introduced the following as super trait of `VarDef`, `Let` and `ParamDef`:
```scala
  sealed trait LocalDef extends Tree {
    def name: Ident
    def mutable: Boolean

    def ref(implicit pos: Position): Tree = VarRef(name)
  }
```